Apps/Inc/Tasks.h

```diff
@@ -135,4 +135,7 @@ extern OS_SEM SendCarCAN_Sem4;
  */
 // TODO: Put all global state variables here
 
+extern car_state_t carState;
+
+
```

Comment:
This doesn't need to be global. It'll be passed to each task in `main()` as a `p_arg`